src/analysis/retail/paladin/retribution/modules/Abilities.tsx

```diff
@@ -129,7 +130,8 @@ class Abilities extends CoreAbilities {
         gcd: {
           base: 1500,
         },
-        castEfficiency: {
+        enabled: !combatant.hasTalent(TALENTS_PALADIN.CONSECRATED_BLADE_TALENT),
+        castEfficiency: combatant.hasTalent(TALENTS_PALADIN.CONSECRATED_BLADE_TALENT) ? {} : {
```

Comment:
I couldn't find an example for this kind of functionality. Generic cast efficiency does not like _only_ setting enabled while leaving `castEfficiency` alone